src/GraphBLAS-sharp.Backend/Algorithms/Algorithms.fs

```diff
@@ -19,3 +19,8 @@ module Algorithms =
 
     module SSSP =
         let run = SSSP.run
+
+    module PageRank =
+        let run = PageRank.run
```

Comment:
I think documentation is needed. At least we should say that matrix has to be prepared first by calling prepareMatrix method.